yarn.lock

```diff
@@ -1106,7 +1106,7 @@
     exec-sh "^0.3.2"
     minimist "^1.2.0"
 
-"@commitlint/cli@^7.1.2", "@commitlint/cli@^7.6.1":
+"@commitlint/cli@^7.6.1":
   version "7.6.1"
```

Comment:
This PR doesn't change anything, because the yarn.lock file already indicated that 7.6.1 should be loaded.